packages/e2e-tests/config/setup-test-framework.js

```diff
@@ -187,28 +187,34 @@ async function simulateAdverseConditions() {
 		return;
 	}
 
-	const client = await page.target().createCDPSession();
+	if ( OFFLINE ) {
+		await page.setOfflineMode( true );
```

Comment:
The docs aren't very clear, but I this this is needed if we ever want to test a PWA. But I don't fully know what it does.